packages/repo-metadata-lint/README.md

```diff
@@ -0,0 +1,22 @@
+# repo-metadata-lint
+
+Validates the fields in `.repo-metadata.json`:
```

Comment:
Adding an `install` link is always a kindness here. Is there a doc that explains the metadata schema?  May be nice to outline here or point to that doc.